welcome/index.adoc

```diff
@@ -62,12 +74,13 @@ Start with **xref:../architecture/architecture.adoc#architecture-overview-archit
 endif::[]
 
 ifdef::microshift[]
-Start with xref:../microshift_getting_started/microshift-understanding.adoc#microshift-understanding[Understanding {product-title}] and xref:../microshift_install/microshift-install-rpm.adoc#microshift-install-rpm[Installing].
-Then, see the xref:../microshift_release_notes/microshift-4-13-release-notes.adoc#microshift-4-13-release-notes[release notes].
-
 [IMPORTANT]
 ====
-Red Hat does not provide or support an update or upgrade path from the Technology Preview version to later versions of {product-title}. A new installation is necessary.
+{product-title} is Technology Preview only. This Technology Preview software is not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using {product-title} in production. Technology Preview provides early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
```

Comment:
There is a TP snippet. I suggest using the snippet. In the event our language around TP changes, you will pick up the update.

Reply:
Unfortunately it's not quite accurate for MicroShift. This is most of it with some tweaks.